traxxas_slash_4x4_platinum_2012/VehicleConfigParams.yaml

```diff
@@ -0,0 +1,94 @@
+# Defines the ros parameters which define the characteristics of this host vehicle configuration
+# The value type field is used to indicate how the field should be set. 
+# Should it be treated as a measured value (Measured) or a desired behavior constraint (Desired)
+
+# String: Host vehicle make
+# Value type: Measured
+vehicle_make: 'Traxxas'
+
+# String: Host vehicle model
+# Value type: Measured
+vehicle_model: 'Slash 4X4 Platinum'
+
+# Integer: Host vehicle year
+# Value type: Measured
+# Units: Year
+vehicle_year: 2012
+
+# Double: Host vehicle length
+# Value type: Measured
+# Units: Meters
+vehicle_length: 0.568
+
+# Double: Host vehicle width
+# Value type: Measured
+# Units: Meters
+vehicle_width: 0.296
+
+# Double: Host vehicle height
+# Value type: Measured
+# Units: Meters
+vehicle_height: 0.193
+
+# Double: Distance from front axel to rear axel
+# Value type: Measured
+# Units: Meters
+vehicle_wheel_base: 0.324
+
+# Double: Radius of the tires
+# Value type: Measured
+# Units: Meters
+vehicle_tire_radius: 0.055
+
+# Acceleration limit
+# Value type: Desired
+# Units: m/s^2
+# vehicle_acceleration_limit: 
+
+# Deceleration limit
+# Value type: Desired
+# Units: m/s^2
+# vehicle_deceleration_limit: 
+
+# Lateral Acceleration Limit
+# Value type: Desired
+# Units: m/s^2
+# vehicle_lateral_accel_limit: 
```

Comment:
These acceleration limits are needed by guidance. Do you have a plan for setting them from a different source?

Reply:
I haven't needed them yet, so I haven't bothered to figure out what the values need to be. Once I know, I will update them. We can hold off on the PR until then if you would like. I'm making do with using my fork.

Reply:
I think you should add them back so that we don't get confused later looking for missing parameters.

Reply:
I based this off the config file from `ford_fusion_sehybrid_2019`, so the would be incorrect if I put them back. If we need to have values there, I can set them as `0.0` to have some safe defaults.

Reply:
Zero is fine for now